### ISSUE TITLE
[Accessibility] Export Page: Element has low color contrast

### ISSUE THREAD

**Issue by nwanduka** [camicroscope/caMicroscope#1034]
**Describe the bug**
The small text underneath the "Export Results" heading doesn't have sufficient color contrast.

**To Reproduce**
1. Go to 'Export' in the nav bar
2. Run the Axe a11y test or Lighthouse
3. See error

**Expected behavior**
Elements should have sufficient color contrast.

**Screenshots**
![image](https://github.com/user-attachments/assets/3c106571-80f1-474f-82a9-02534aaf1881)

**Comment by Priyankagarai** [comment]
Hi @nwanduka I hope you’re doing well. I’ve created a pull request for the low color contrast issue Could you please take some time to review it?
I would appreciate it if you could provide any feedback or suggestions. Once reviewed, could you also merge it if everything looks good?
